docs/advanced-topics/metrics.rst

```diff
@@ -0,0 +1,154 @@
+
+Go to https://dockstore.org/api/static/swagger-ui/index.html#/extendedGA4GH/executionMetricsPost. This is the endpoint used to submit metrics to Dockstore. Click the "Try it out” button.
+
+First, fill out the query parameters. This is where you'll input information about which workflow was executed and what platform it was executed on.
+
+For the ``id`` parameter, provide the TRS ID of the workflow that you want to submit metrics for. For example, the `dockstore-tool-bamstats <https://dockstore.org/workflows/github.com/dockstore/dockstore-tool-bamstats/wdl:1.25-9?tab=info>`__ workflow has the TRS ID ``#workflow/github.com/dockstore/dockstore-tool-bamstats/wdl`` as shown in the “Info” tab with the label “TRS".
+
+For the ``version_id`` parameter, provide the name of the workflow version that was executed. It can be any version listed in the "Versions" tab of the workflow. For example, `dockstore-tool-bamstats <https://dockstore.org/workflows/github.com/dockstore/dockstore-tool-bamstats/wdl:1.25-9?tab=versions>`__ has the following versions currently: 1.25-9 and develop. It is recommended to submit execution metrics for versions that are unlikely to change, like tags.
+
+Select the ``platform`` that the workflow was executed on from the available values.
+
+For the ``description``, you may provide an optional description about the metrics you're submitting.
+
+.. figure:: /assets/images/docs/submit-metrics/query-parameters.png
+    :alt: Query parameters for submitting metrics
+
+In the request body, provide the workflow execution metrics that you want to submit. Click on Schema to view the schema of the request body.
+
+.. figure:: /assets/images/docs/submit-metrics/request-body-schema.png
+    :alt: Request body schema for submitting metrics
+
+You can provide a individual executions through ``runExecutions`` and ``validationExecutions``.
+
+A run execution is an execution that runs the workflow. The only required metric for run executions is ``executionStatus``. If you want to provide additional metrics that are not defined in the schema, use the ``additionalProperties`` key to provide your metric.
+
+.. figure:: /assets/images/docs/submit-metrics/run-executions-schema.png
+    :alt: Schema for run executions
+
+A validation execution is an execution of a validator tool, like miniwdl, on the workflow. The following are required for a validation execution: 
+
+- ``validatorTool``
+- ``validatorToolVersion``
+- ``isValid``
+- ``dateExecuted``
+
+If you want to provide additional metrics that are not defined in the schema, use the ``additionalProperties`` key to provide your metric.
+
+.. figure:: /assets/images/docs/submit-metrics/validation-executions-schema.png
+    :alt: Schema for validation executions
+
```

Comment:
What  is an aggregated metric? 
i.e. is this covered elsewhere already? otherwise, may need to introduce it here